src/document.h

```diff
@@ -1312,6 +1343,7 @@ struct Document {
                     c->Paste(this, sys->cellclipboard.get(), selected);
                     Refresh();
                 }
+                pastect = false;

```

Comment:
why is this not just an argument to `PasteOrDrop` ?

Reply:
Thanks for this hint! I made it an optional argument to PasteOrDrop.